### PR TITLE
SAK-47362 Content tool: Error uploading pasted images in editor

### PR DESCRIPTION
**Pull request by ropemar**
Fixed a problem when pasting images in the CK editor. When you reached 100 images pasted and then uploaded, you got an error because all images were named the same ("image"). It was being considered as a hundredth attempt to upload the same image, but it was not.

In addition, I have added internationalization of error messages related to these uploads.

Please @josecebe @mpellicer, could you keep an eye out?

### CODE REVIEW

File: content/content-tool/tool/src/java/org/sakaiproject/content/entityproviders/ContentEntityProvider.java [hunk 1]
```diff
@@ -116,6 +118,9 @@ public class ContentEntityProvider extends AbstractEntityProvider implements Ent
 	private UserDirectoryService userDirectoryService;
 	private EntityManager entityManager;
 	private ServerConfigurationService serverConfigurationService;
+	
+	/** Resource bundle */
+    private static ResourceLoader rb = new ResourceLoader("content");
```
Comment (jesusmmp) [inline content/content-tool/tool/src/java/org/sakaiproject/content/entityproviders/ContentEntityProvider.java:123]: Spacing

Comment (ropemar) [reply content/content-tool/tool/src/java/org/sakaiproject/content/entityproviders/ContentEntityProvider.java]: Yes, missed that, thanks!

**Review by jesusmmp — APPROVED** [review]
LGTM!

**Review by mpellicer — APPROVED** [review]
Awesome!

**Pull request merged**